README.md

```diff
@@ -1,8 +1,19 @@
-# tree-sitter-sourcepawn
-
-[![npm version](https://badge.fury.io/js/tree-sitter-sourcepawn.svg)](https://badge.fury.io/js/tree-sitter-sourcepawn)
-
-[sourcepawn](https://github.com/alliedmodders/sourcepawn) grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter)
+<div align="center">
+  <h1><code>tree-sitter-sourcepawn</code></h1>
+  <p>
+    <strong>[SourcePawn](https://github.com/alliedmodders/sourcepawn) grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter)</strong>
```

Comment:
```suggestion
    <strong><a href="https://github.com/alliedmodders/sourcepawn" target="_blank">SourcePawn</a> grammar for <a href="https://github.com/tree-sitter/tree-sitter" target="_blank">tree-sitter</a></strong>
```
Markdown links won't work in HTML code. So converting this to `<a>` :slightly_smiling_face:

Reply:
Woups, looks like I went way too fast on the copy paste 🐇